h3/bq/sql/H3_ISPENTAGON.sql

```diff
@@ -0,0 +1,24 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_H3@@.__H3_ISPENTAGON`(index_lower INT64, index_upper INT64)
+    RETURNS BOOLEAN
+    DETERMINISTIC
+    LANGUAGE js
+    OPTIONS (library=["@@H3_BQ_LIBRARY@@"])
+AS
+"""
+    if (index_lower == null || index_upper == null)
+        return false;
```

Comment:
Here we are not checking if is valid neither.

Reply:
Same, and it's covered by the integration tests.